src/main/groovy/io/seqera/wave/configuration/BuildConfig.groovy

```diff
@@ -111,4 +111,19 @@ class BuildConfig {
     String getSingularityImageArm64(){
         return singularityImageArm64 ?: singularityImage + "-arm64"
     }
+
+    // check this link to know more about these options https://github.com/moby/buildkit/tree/master/examples/kubernetes#kubernetes-manifests-for-buildkit
+
+    Map<String,String> getEnvironment() {
+        return  ['BUILDKITD_FLAGS': '--oci-worker-no-process-sandbox']
+    }
+
+    String getBuildCommand(){
+        return 'buildctl-daemonless.sh'
+    }
+
+    Map<String, String> getAnnotations(String containerName) {
+        final key = "container.apparmor.security.beta.kubernetes.io/$containerName".toString()
+        return Map.of(key, "unconfined")
+    }
```

Comment:
Those don't look to be real config params. It would make more sense to move these in the k8s services using it

Reply:
k8s service is responsible for build, scan and transfer, but these properties are related to build only.
So I think it will be easier to keep build-related properties in one class

Reply:
Yeah but `BuildConfig` is meant to hold config attributes and those are not. `getBuildCommand` should go [BuildStrategy](https://github.com/seqeralabs/wave/blob/9b39688972699334f302ddb8ad655273328e809b/src/main/groovy/io/seqera/wave/service/builder/BuildStrategy.groovy#L33-L33) because it's used both by docker and K8s. 

`getAnnotations` and `getEnvironment` can even be literals to make more explicit the reason for which they are used